blocks/library/paragraph/index.js

```diff
@@ -377,20 +363,21 @@ export const settings = {
 			dropCap,
 			backgroundColor,
 			textColor,
-			textClass,
 			fontSize,
 		} = attributes;
 
+		const thresholdFontSize = findKey( FONT_SIZES, ( size ) => size === fontSize );
```

Comment:
One thing here is that we'd want this to be abstracted away from blocks when we make FontSize a component.

Reply:
Maybe this could be a filterLike function `props, fontSize => updateProps`

`const props = addFontSizeProps( { style, className } )`